edx_event_bus_kafka/internal/consumer.py

```diff
@@ -114,6 +115,7 @@ def consume_indefinitely(self):
             self.consumer.close()
             logger.info("Committing final offsets")
 
+    @background_task()
```

Comment:
@dianakhuang: I know this merged, but some things to consider:

1. Should the `poll` call be included in the task? Do we need a method that wraps all of the following?:
```
    msg = self.consumer.poll(timeout=CONSUMER_POLL_TIMEOUT)
    if msg is not None:
        self.process_single_message(msg)
```
2. Will all tasks have the same name? Do we need to use a context manager to set the task name, possibly using `msg.topic()` or something?
3. How can we convey when there was an error to New Relic? Do we have enough info from `msg.error()` to use `notice_error`? See https://docs.newrelic.com/docs/apm/agents/python-agent/python-agent-api/noticeerror-python-agent-api/? Note: edx-django-utils currently exposes `record_exception`, which has been deprecated in favor of notice_error. We may want to mark is as deprecated and add the new method.